alignment_comment.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#Description:
+#Script aligns fastq files in zipped format with HISAT2
+#Converts Sam output into BAM
+#Then creates BAM Index Files for visualization (.bai)
+
+#Written & Maintained by Vasco Morais (April 2019)
+
+echo $RNA_HOME
+
+export RNA_PRACTICE_DATA_DIR=$RNA_HOME/practice
+echo $RNA_PRACTICE_DATA_DIR
+
+RNA_PATH=$RNA_HOME/practice
+ADAPTER=${RNA_REFS_DIR}'/illumina_multiplex.fa'
+MIX='_r'
+OUTPUT='_Build37-ErccTranscripts-chr22'
+
+echo $RNA_PATH
+echo $ADAPTER
+
+
+echo '-------------------------------'
+echo 'BEGIN SCRIPT'
+
+for m in 1 2
+do
+if [ $m = 1 ]; then
+	REP='hcc1395_normal_rep'
+	ID='HCC1395_normal_rep'
+	OUTPUT='HCC1395_normal.bam'
+elif [ $m = 2 ]; then
+	REP='hcc1395_tumor_rep'
+	ID='HCC1395_tumor_rep'
+	OUTPUT='HCC1395_tumor.bam'
+fi
+for r in 1 2 3;
+do echo 'Mix: '${m};
```

Comment:
Use indentation without a semicolon for example 
`
for i in file1 file2 file3
do
   command1
    command2
`
or
`
for i in file1 file2 file 3; do command1 command2; done
`